students/dabeen/westagram/users/views.py

```diff
@@ -1,3 +1,77 @@
 from django.shortcuts import render
 
+import json
+import bcrypt
+import re
+import jwt
+from django.http import JsonResponse
+from django.views import View
+from users.models import User
+
 # Create your views here.
+
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            email = data['email']
+            password = data['password']
+
+            # Return 400 error when using an existing email.
+            if User.objects.filter(email=email).exists():
+                return JsonResponse({'message': 'EMAIL_ALREADY_EXISTS'}, status=400)
```

Comment:
자원이 존재하지 않는 경우 status code로 404를 반환해주세요~
```suggestion
            if User.objects.filter(email=email).exists():
                return JsonResponse({'message': 'EMAIL_ALREADY_EXISTS'}, status=400)
```